src/Serilog.Sinks.MSSqlServer/Sinks/MSSqlServer/MSSqlServerSink.cs

```diff
@@ -332,8 +330,21 @@ private string LogEventToJson(LogEvent logEvent)
                 var columnName = property.Key;
                 var columnType = row.Table.Columns[columnName].DataType;
                 object conversion;
+
                 var scalarValue = property.Value as ScalarValue;
-                if (scalarValue != null && TryChangeType(scalarValue.Value, columnType, out conversion))
+                if (scalarValue == null)
+                {
+                    row[columnName] = property.Value.ToString();
```

Comment:
Thanks for the PR!

Is the column necessarily going to accept a string here? Seems like the result of `ToString()` might need to go through `TryChangeType()` - is that the case? Cheers!